Update 3 projects to use new webhooks (#60)
autokitteh/kittehub: google_forms_to_jira/README.md

# Google Forms to Jira

This project creates Jira issues based on Google Forms responses.

## How It Works

> 1. The workflow is triggered by response events from Google Forms, for a
>    specific form ID

Reviewer: ```suggestion
1. The workflow is triggered by response events from a specific Google Form, identified by its unique form ID
```
